### DISCUSSION TITLE
Azure Arc

### DISCUSSION THREAD

**Discussion by Sylve32** [weeyin83/Azure-Arc-Windows-Linux-Dashboard#6] [Q&A] [answered]
Hi Sarah, 
I have a query that shows up the number of cores per each server and I am presenting it in a pie chart view (donut view) and in the middle of the donut its computing the percentage cores of each server. Can I have a query that computes the number of cores per server not the percentage

Regards

**Comment by weeyin83** [comment]
Are you able to share the query?

**Reply by Sylve32** [reply]
Yes sure
please find below

resources
| where type=~ 'microsoft.hybridcompute/machines'
| extend CoreCount = toint (properties.detectedProperties.logicalCoreCount)
| project name, CoreCount
|summarize totalCores = sum(CoreCount) by name

**Comment by weeyin83** [comment, marked as answer]
When you display the results within a donut chart, it will always show the answer in the middle as a percentage.  If you just want to see the core count you will need to display the results either as a table or a bar chart.

**Comment by Sylve32** [comment]
Yes, so isn't there a way I would show it in a donut view and show the core count in the middle rather than a percentage

**Comment by weeyin83** [comment]
Unfortunately not.
